src/cache/jsongMerge.js

```diff
@@ -65,7 +66,7 @@ function merge(config, cache, message, depth, path, fromParent, fromKey) {
             var values = config.values;
             values.push({
                 path: cloneArray(requestedPath),
-                value: message.type ? message.value : message
```

Comment:
@michaelbpaulson, @jhusain 

The `message.type` here seems to be a bug also, but 'fixing' it (I assume it's supposed to be `message.$type` as opposed to `message.type`) breaks the `JSONG - Merge should write a simple path to the cache` basic unit test in jsongMerge.spec.js. Needed some input on what the expected output should be for this test to determine if the test is broken, or there are potentially more bugs in the jsongMerge impl, which need to be teased out.